docs/topics/upgrade.md

```diff
@@ -42,11 +42,21 @@ In order to ensure that your `aks-engine upgrade` operation runs smoothly, there
 
 During the upgrade, *aks-engine* successively visits virtual machines that constitute the cluster (first the master nodes, then the agent nodes) and performs the following operations:
 
+Master nodes:
```

Comment:
Same question I asked in the PR thread here:
What about master nodes? I realize that this feature was motivated by AKS customer requests and AKS doesn't have master but if this is going to be an aks-engine first class feature we should apply the change for master nodes as well. Otherwise aks-engine users will have to deal with inconsistency of their agent nodes preserving tags during upgrades but not master nodes.

Reply:
add this new behavior to master nodes too.

Reply:
Thanks! Can we add it to the doc here for master as well? Do we still need 2 separate sections for agents and masters?

Reply:
oh, thanks for mentioning it. I forget to update the doc in my previous push. We still need 2 separate section for masters and agents because they are upgraded in different order
1) master: delete one old node, add a new node, 
2) agent:   add a new node, delete one old node.